cobraproclimits/cobraproclimits.go

```diff
@@ -26,7 +27,11 @@ func SetMemLimitRunE(options ...memlimit.Option) cobrautil.CobraRunFunc {
 			logLevelMap[zLevel] = sLevel
 		}
 
-		logger := zerolog.DefaultContextLogger
+		noop := zerolog.Nop()
```

Comment:
Do we want this to be a nop logger or the default?

Reply:
Personally, I'm not too concerned about not logging those because the go runtime exposes metrics around gomaxprocs and gomemlimits, but we can add the default logger if you'd like